changelogs/unreleased/gh-10489-fix-vinyl-on-dump-failure-memleak.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/vinyl
+
+- Fixed a memory leak in vinyl on dump/compaction failure (gh-10489).
```

Comment:
"in vinyl" is redundant here - this changelog entry is for Vinyl.

Reply:
Fixed as suggested.